templates/puppetagent.conf.erb

```diff
@@ -53,9 +53,12 @@
 <% if @http_proxy_port != 'UNSET' -%>
     http_proxy_port = <%= @http_proxy_port %>
 <% end -%>
-    report = true
+    report = <%= @puppet_reports %>
     graph = true
     pluginsync = true
+    interval = <%= @puppet_run_interval %>
```

Comment:
I think you mean `run_interval` - http://docs.puppetlabs.com/references/latest/configuration.html#runinterval